mdsuite/graph_modules/molecular_graph.py

```diff
@@ -72,7 +77,7 @@ def __init__(
         self.species = species
 
         self.database = Database(
-            name=os.path.join(self.experiment.database_path, "database.hdf5"),
+            name=(self.experiment.database_path / "database.hdf5").as_posix(),
```

Comment:
Is this a feature of Path objects?

Reply:
Yes, unfortunately the Database class does not handle pathlib objects, so we have to convert it back to a posix path.
I just read https://towardsdatascience.com/why-you-should-start-using-pathlib-as-an-alternative-to-the-os-module-d9eccd994745 for a small overview.